superset-frontend/.eslintrc.js

```diff
@@ -16,6 +16,70 @@
  * specific language governing permissions and limitations
  * under the License.
  */
+const rules = {
```

Comment:
This extraction is temporary. Will move it back to make PR easier to review and do the extraction in another PR.